### PR TITLE
Bug fix: showing legend based on grid state

### PR DESCRIPTION
**Pull request by SidSyrus**
- Svg foreign object for track legend icon
- Showing track legend tooltip based on grid state

### CODE REVIEW

File: src/OncoGrid.js [hunk 1]
```diff
@@ -130,6 +130,7 @@ OncoGrid.prototype.resize = function(width, height, fullscreen) {
   var _self = this;
 
   _self.fullscreen = fullscreen;
+  _self.mainGrid.fullscreen = fullscreen;
   _self.charts.forEach(function (chart) {
     chart.fullscreen = fullscreen;
     chart.resize(Number(width), Number(height));
```
Comment (SidSyrus) [inline src/OncoGrid.js:136]: @andricDu  Is this line `_self.fullscreen = fullscreen;` needed? I don't think we are using it.

File: test/page/index.js [hunk 2]
```diff
@@ -92,7 +92,7 @@ var params = {
   width: 600,
   heatMap: true,
   trackHeight: 20,
-  trackLegendLabel: '(Legend)',
+  trackLegendLabel: '?',
```
Comment (andricDu) [inline test/page/index.js:95]: 👍

File: src/OncoGrid.js (hunk 1 above)
Comment (andricDu) [reply src/OncoGrid.js:136]: @SidSyrus is it being used by the portal?

Comment (SidSyrus) [reply src/OncoGrid.js:136]: I removed it and oncogrid still seems to work. However, I am not 100% sure if we are not using it.

**Pull request merged**